packages/web-app-files/src/composables/scrollTo/useScrollTo.ts

```diff
@@ -21,29 +21,25 @@ export const useScrollTo = (): ScrollToResult => {
     return queryItemAsString(unref(detailsQuery))
   })
 
-  const scrollToResource = (resource) => {
+  const scrollToResource = (resourceId: Resource['id'], forceScroll = false) => {
```

Comment:
Not a fan of bool params tbh, can we make it an options obj? 

```suggestion
  const scrollToResource = (resourceId: Resource['id'], { forceScroll = false}) => {
```